data/base-spells.yaml

```diff
@@ -227,6 +227,12 @@ spell_data:
     ability: Targeted Magic
     abbrev: IZ
     cyclic: true
+  Mind Shout:
+    ability: Debilitation
+    abbrev: MS
+    mana: 20
```

Comment:
I would not put `mana` in `base-spells`. The rest of these attributes are fixed, but mana can change.